remme/clients/pub_key.py

```diff
@@ -15,12 +15,14 @@
 
```

Comment:
The idea is, that clients can still work with certificates. But they should pass new payloads to handlers. 
So basically, to support old internal api, we should leave certificate methods, but refactor their internal implementation.
And in addition create new pub_key client. 
@eugene-babichenko  what do you think about it ?

Reply:
I think it is too early to create a separate client. I will create a separate task for that and we will do it when we will have time or it will become crucial.